app/scripts/modules/amazon/src/serverGroup/details/sections/MultipleInstanceTypesSubSection.tsx

```diff
@@ -0,0 +1,40 @@
+import { IAmazonLaunchTemplateOverrides } from 'amazon';
+import React from 'react';
+
+import { CollapsibleSection, HelpField } from '@spinnaker/core';
+import './multipleInstanceTypesSubSection.less';
```

Comment:
I renamed the css to the specific component name as per suggestions. 
This might need to be renamed in the future as it finds more uses, fyi.